lobby/build.gradle

```diff
@@ -0,0 +1,182 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'org.junit.platform:junit-platform-gradle-plugin:1.0.2'
+    }
+}
+
+plugins {
+    id 'java'
+    id 'application'
+    id 'checkstyle'
+    id 'jacoco'
+    id 'com.github.ben-manes.versions' version '0.17.0'
+    id 'com.github.johnrengelman.shadow' version '2.0.2'
+    id 'com.install4j.gradle' version '7.0.1'
+    id 'de.undercouch.download' version '3.3.0'
+}
```

Comment:
I don't know that much about gradle, but I'm pretty sure there's a way this duplication as well as the duplication of tasks in this project can be avoided.

Reply:
There is, and it is a good concern. My plans for it are to wait a bit before we clean up the duplication. I think waiting on this is okay for a few reasons:
1. sub-projects in flux, there could be churn to de-dupe things and then repeat that work to keep it all perfectly de-duped
2. build duplication is not that bad; it *is* a different league compared to code duplication. So we have quite some time before build configuration duplication would really hurt us. 
3. first focused on getting split to work

with that said, I'll take a poke to see what kind of low hanging fruit there are, but I do want to focus elsewhere and also see how the build scripts shape up a bit first before trying to make things common.